### ISSUE TITLE
cannot find audio device config on new installs of NVDA

### ISSUE THREAD

**Issue by fastfinge** [mltony/nvda-phonetic-punctuation#25]
On new versions of NVDA, the audio device config was moved from "speech" to "audio" in NVDA.ini. You need to update this in two places in commands.py or else the addon won't work in fresh installs of NVDA.

**Comment by fastfinge** [comment]
Oh, I use the alphas on my testing virtual machine. This was a breaking change just made today. Never mind.
